torch/csrc/jit/passes/quantization.cpp

```diff
@@ -145,6 +146,32 @@ Node* createIntReprNode(Value* v, Graph* g) {
   return intrepr;
 }
 
+// Create observer.forward Node and insert a call to observer forward function
+Node* insertObserverForwardCall(Value* v, Graph* g, script::Module module, script::Module observer_module) {
+  std::string observer_name = "observer_for_" + v->debugName();
+  module.register_module(observer_name, observer_module.clone());
+  WithInsertPoint ins(v->node());
```

Comment:
You don't need to use `WithInsertPoint` if you use `insertAfter`. `WithInsertPoint` only affects `insert` function, `insertBefore` and `insertAfter` explicitly specify where the node should be inserted.